pom.xml

```diff
@@ -821,6 +810,9 @@
         <org.jacoco.version>0.7.9</org.jacoco.version>
         <carbon.feature.plugin.version>3.1.3</carbon.feature.plugin.version>
         <maven.exec.plugin.version>1.5.0</maven.exec.plugin.version>
+        <maven.clean.plugin.version>3.0.0</maven.clean.plugin.version>
+        <maven.clean.plugin.version>3.0.0</maven.clean.plugin.version>
```

Comment:
I think you have mistakenly added the same version

Reply:
Fixed in e657224